monitor/monitor.go

```diff
@@ -174,4 +225,8 @@ func (m *Monitor) Run() {
 	if m.submitter != nil {
 		m.submitter.run()
 	}
+
+	if m.inclusionChecker != nil {
+		m.inclusionChecker.run()
```

Comment:
When `m.inclusionChecker` has a `stop()` it will need to be wired into `monitor.Stop()` as well.